ONSdigital/design-system
# Remove website generation

### What is the context of this PR?

This pr removes the website build process from the DS as all documentation now lives in the service manual. 

### How to review
- Check to see if any other logic can be removed from the design system.
- Verify that build/preview commands now shows a listing of examples instead of documentation pages.
- Verify that unit tests pass.
